CONTRIBUTING.md

```diff
@@ -467,6 +467,14 @@ except (ImportError, IOError) as err:
 This rule doesn't apply to the test files since these files are loaded only when
 running tests and in such a case your module and their external dependencies are installed.
 
+**You also need to test your code without the python lib nor the module installed** 
```

Comment:
Hmmmm I think I might agree with @dreispt on the feasibility that this is actually going to be done - that's a decent amount of overhead to my less ops-oriented engineers.

Just so I'm making sure I understand the intent - it's to make sure that there are not any external library calls or whatever in the global namespace?

If this is the case, I feel like this is actually a case we can watch for in lint. @moylop260 do you have any thoughts on that?